AdvanceSwift.playground/Pages/Class.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,215 @@
+import UIKit
+//1
+class Demo {
+    var id: Int
+    var name: String
+    init(id: Int,name: String) {
+        self.id = id
+        self.name = name
+    }
+}
+let demo = Demo(id:1,name:"Stuti")
+demo.id = 3  //set
+print(demo.id)  //get access
+print(demo.name)
+print()
+
+//2 multiple initialize
+print("2. Multiple Initializer")
+class Person {
+    var firstName: String?
+    var clg: String?
+    var dept: String?
+    
+    init(firstName: String, clg: String) {
+        self.firstName = firstName
+        self.clg = clg
+    }
+    
+    init(firstName: String, dept: String) {
+        self.firstName = firstName
+        //self.clg = clg
+        self.dept = dept
+    }
+}
+
+let person = Person(firstName: "Stuti", clg:"VGEC")
+print(" First name \(person.firstName!), clg: \(person.clg!) ")
+
+let person3 = Person(firstName: "Stuti", dept: "CE")
+print(" First name \(person3.firstName!), Dept: \(person3.dept!) ")
+
+//3 without initializer
+print()
+print("Without Initializer")
+class InitDemo {
+    var id : Int = 0
+    var name : String? = ""
+}
+
+var demo1 = InitDemo()
+demo1.id = 1
+demo1.name = "Stuti"
+print("\(demo1.id) \(demo1.name!)")
+
+print()
+//4 squarer of number
+print("4. Square of the number")
+class Square {
+     func squareNumber(number: Int) -> Int {
+        return number * number
+    }
+}
+var a = Square()
+print(a.squareNumber(number: 2))
+
+//5 inheritance
+print()
+print("5. Swift Inheritance")
+class Coder {
+      var favouriteEditor = "vi"
+      init() {}
+      func startEditorWar() {
+          print("Obviously \(favouriteEditor) is best")
+      }
+  }
+ 
+  class Rubyist:Coder {
+     override init(){
+         super.init()
+         favouriteEditor = "TextMate"
+    }
+ }
+
+ class Lisper:Coder {
+     override init(){
+         super.init()
+        favouriteEditor = "emacs"
+     }
+ }
+
+ var r = Rubyist()
```

Comment:
variable name should be meaning ful, 'r', 'l' is not allowed. check this in whole file